installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -61,11 +62,12 @@ After you deploy your cluster, you can attach pods to additional networks. For m
 include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+1]
 include::modules/installation-osp-converting-ignition-resources.adoc[leveloffset=+1]
 include::modules/installation-osp-creating-control-plane-ignition.adoc[leveloffset=+1]
-include::modules/installation-osp-creating-network-resources.adoc[leveloffset=+1]
+include::modules/installation-osp-updating-network-resources.adoc[leveloffset=+1]
 
 Optionally, you can use the `inventory.yaml` file that you created to customize your installation. For example, you can deploy a cluster that uses bare metal machines.
```

Comment:
I noticed this was hanging out at the end of the "Updating network resources on RHOSP" section, after the procedure steps. Just checking whether this is still intentional after the latest changes. Could be fine as-is, or maybe it works as either a `[NOTE]` or a final `. Optional:` step in the new procedure?

Reply:
Great catch, thank you! I'll move it as a final step in the procedure.